cmd/zfs/zfs_main.c

```diff
@@ -8114,6 +8114,15 @@ main(int argc, char **argv)
 	    (strcmp(cmdname, "--help") == 0))
 		usage(B_TRUE);
 
+	/*
+	 * Special case '-V|--version'
+	 */
+	if ((strcmp(cmdname, "-V") == 0) ||
+	    (strcmp(cmdname, "--version") == 0)) {
```

Comment:
Seems like this should appear in the usage message as well.  How to do that depends on if we convert this to a subcommand or not.

Reply:
I would defer to yourself and @behlendorf on making that determination.

Reply:
The usage doesn't show `--help`, so I don't think it needs `-V` or `--version` either. It seems like the "main" way is the `version` subcommand, which is listed in the usage. So I think this is fine as is, but I'm not going to go so far as to mark this resolved.